src/game/BattleGround.cpp

```diff
@@ -374,6 +372,18 @@ void BattleGround::SendPacketToTeam(uint32 TeamID, WorldPacket *packet, Player *
     }
 }
 
+void BattleGround::SendPacketToEnemyTeam(uint32 TeamID, WorldPacket *packet)
+{
+	for (BattleGroundPlayerMap::iterator itr = m_Players.begin(); itr != m_Players.end(); ++itr){
+        if (Player *plr = sObjectMgr.GetPlayer(itr->first)){
+			uint32 team = GetPlayerTeam(plr->GetGUID());
+			if (team && team != TeamID){
```

Comment:
Maybe I'm wrong but from my point of view there is no need to pass enemy teamid since we have to iterate twice also to get the player object, too.